cloudsplaining/command/scan_multi_account.py

```diff
@@ -0,0 +1,205 @@
+"""Scan multiple AWS accounts via AssumeRole"""
+import logging
+import os
+import json
+import yaml
+import click
+import boto3
+from click_option_group import optgroup, RequiredMutuallyExclusiveOptionGroup
+from cloudsplaining.shared.constants import EXCLUSIONS_FILE
+from cloudsplaining.command.download import get_account_authorization_details
+from cloudsplaining import set_log_level
+from cloudsplaining.shared.exclusions import Exclusions, DEFAULT_EXCLUSIONS
+from cloudsplaining.shared import utils, aws_login
+from cloudsplaining.shared.validation import check_authorization_details_schema
+from cloudsplaining.scan.authorization_details import AuthorizationDetails
+from cloudsplaining.output.report import HTMLReport
+
+logger = logging.getLogger(__name__)
+
+
+@click.command(
+    short_help="Scan multiple AWS Accounts using a config file"
+)
+@click.option(
+    "--config-file",
+    "-c",
+    type=click.Path(exists=True),
+    required=True,
+    help="Path of the multi-account config file",
+)
+@click.option(
+    "--profile",
+    "-p",
+    type=str,
+    required=False,
+    help="Specify the AWS IAM profile.",
+    envvar="AWS_PROFILE"
+)
+@click.option(
+    "--role-name",
+    "-r",
+    type=str,
+    required=True,
+    help="The name of the IAM role to assume in target accounts. Must be the same name in all target accounts."
+)
+@click.option(
+    "--exclusions-file",
+    help="A yaml file containing a list of policy names to exclude from the scan.",
+    type=click.Path(exists=True),
+    required=False,
+    default=EXCLUSIONS_FILE,
+)
+@optgroup.group(
+    "Output Target Options",
+    cls=RequiredMutuallyExclusiveOptionGroup,
+    help="",
+)
+@optgroup.option(
+    "--output-directory",
+    "-o",
+    "output_directory",
+    required=False,
+    type=click.Path(exists=True),
+    # default=os.getcwd(),
+    help="Output directory. Supply this or --bucket.",
+)
+@optgroup.option(
+    "--bucket",
+    "-b",
+    "save_bucket",
```

Comment:
🎨  Even though this is an internal variable, I think it can be renamed for clarity. Perhaps `output_bucket`? I had to check the type hint again because when reading `if save_bucket:` I thought it was a boolean.

Reply:
Renamed in the most recent commit. Great suggestion.